ynput/ayon-core
# Deadline: adding Nuke into pool collector

## Changelog Description
Adding nuke into Deadline Pool collector. Also enabling the collection on prerender product type.


## Additional info:
This PR is solving the attempt form https://github.com/ynput/ayon-core/pull/446.

## Testing notes:
1. configure your pool settings in `ayon+settings://deadline/publish/CollectDeadlinePools`
2. Open nuke and publish Render or Prerender product type
3. Notice that two new plugin attributes are offered for Primary and Secondary pool selection.
